.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: If rxjs does not behave as expected
+
+---
+
+## Bug Report
+
+**Current Behavior**
+A clear and concise description of the behavior.
+
+**Reproducible code**
```

Comment:
"Reproduction"

we want a minimal reproduction. But sometimes problems aren't just code (build and bundling issues for example)